R/html-tweak.R

```diff
@@ -94,6 +94,35 @@ tweak_all_links <- function(html, pkg = pkg) {
 
   invisible()
 }
+
+
+tweak_navbar_links <- function(html, pkg = pkg) {
+
+  if(!grepl("/$", pkg$meta$url)) {
```

Comment:
I think we should assume no trailing `/`? Or at least that responsibility belongs else where, not specifically in `tweak_navbar_links()`.